frameworks/hdfs/src/main/dist/modify-keytab.sh

```diff
@@ -0,0 +1,3 @@
+export JAVA_HOME=$(ls -d $MESOS_SANDBOX/jdk*/)
+$JAVA_HOME/bin/java -jar krbtest.jar secrets-hdfs.keytab 
```

Comment:
Can we make the keytab an argument for this script?

Reply:
Done